### ISSUE TITLE
"protect" specific folders within a repo

### ISSUE THREAD

**Issue by bebatut** [galaxyproject/galaxy_codex#148]
As the project grows, it might be good to ‘protect’ specific folders within a repo, the way the GTN protects subfolders and also to give specific powers to specific groups (e.g. @nomadscientist  would like Merge/review powers on specifically the single-cell subdomain folders, but not other folders)

**Comment by nomadscientist** [comment]
Yep - and Cameron Hyde @neoformit should have permissions on the new subdomains folder
